Xwt.WPF/Xwt.WPFBackend/ExTreeViewItem.cs

```diff
@@ -226,6 +226,11 @@ private void OnLoaded (object sender, RoutedEventArgs routedEventArgs)
 			}
 		}
 
+		public void ExpandColumnWidth (int columnIndex, double toWidth)

```

Comment:
```suggestion
		public void SetColumnWidth (int columnIndex, double toWidth)
```
and does this need to be `public`?